public/app/features/explore/state/selectors.ts

```diff
@@ -28,3 +31,43 @@ export const deduplicatedLogsSelector = createLodashMemoizedSelector(
     return dedupLogRows(filteredData, dedupStrategy);
   }
 );
+
+const parseRawTime = (value): Moment | string => {
```

Comment:
I think both parseRawTime and timeRangeFromUrlSelector does not belong to state/selectors.ts file. Selectors are basicaly functions that selects slices of redux state, while these two functions feel like utils to me.

Reply:
Fixed. Will push soon